ext/rotoscope/rotoscope.c

```diff
@@ -215,17 +217,18 @@ static void event_hook(VALUE tpval, void *data)
     return;
 
   rs_tracepoint_t trace = extract_full_tracevals(trace_arg, &trace_path);
-  gzprintf(config->log, RS_CSV_FORMAT, RS_CSV_VALUES(trace));
+  fprintf(config->log, RS_CSV_FORMAT, RS_CSV_VALUES(trace));
 }
 
-static void close_gz_handle(Rotoscope *config)
+static void close_log_handle(Rotoscope *config)
 {
   if (config->log)
   {
     if (!in_fork(config))
     {
-      gzclose(config->log);
+      fclose(config->log);
     }
+    close(fileno(config->log));
```

Comment:
Ah, so the FD is shared, but where it's a different PID it's considered a different FD and we should be closing it. Makes sense 👍 

I'm thinking in the parent process we don't need this though, right? Would this make sense:
```c
if (in_fork(config))
{
  close(fileno(config->log));
}
else
{
  fclose(config->log);
}
```